src/EventListener/Translation.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\ContentApiBundle\EventListener;
+
+final class Translation
```

Comment:
Not sure this is a good name (more of a translation request).

Reply:
Can't find a standard name for this, so `TranslationRequest` could do